masonry/src/doc/mod.rs

```diff
@@ -15,6 +15,11 @@
 // These docs all use the .rustdoc-hidden trick described in
 // https://linebender.org/blog/doc-include/
 
+/// Contains the items implemented in "Creating a new widget" and other tutorials.
+pub mod color_rectangle;
```

Comment:
Should this be `pub`? At the very least, the `missing_docs` expect should be removed, because these are now explorable through the public API docs.

My preference would probably lean towards making it an `example`, rather than.
We could use something like:
```
[[examples]]
name = "color_rectangle"
path = "src/doc/color_rectangle.rs"
```
or just stick it in the examples folder.

Alternatively, this can be `doc(hidden)`.